setup.cfg

```diff
@@ -30,23 +30,23 @@ python_requires = >=3.7
 zip_safe = False
 install_requires =
     Flask-Alembic>=2.0.1
-    Flask-SQLAlchemy>=2.1,<3.0.0
+    Flask-SQLAlchemy>=3.0,<4.0.0
     invenio-base>=1.2.10
     SQLAlchemy-Continuum>=1.3.12
     SQLAlchemy-Utils>=0.33.1,<0.39
```

Comment:
Shall we try to remove the upper pin here?

Reply:
Normally i would say yes, we should remove the upper pin, but i would like to ask you, how do we see as early as possible if the version 4.0.0 creates problems? the github actions on this packages will be stopped after some time. So we will see the problems on a local installation, which could block work for some persons and we have to upper pin it again. 

Otherwise if we use the upper pin, we maybe see some DeprecationWarnings and we could do the same as now and have more time to not block others.

So for me, we should first see if we have a setup where we can see problems in advance before we remove the upper pin.